### PR TITLE
react-next and moved docs cleanup

### LINKED ISSUE #15152: Merge OUFR and react-next into react
This is part 1/4 in a series of changes that will happen in master *after* v7 development moves to a branch, but *before* beta publishing:
1. (this) #15152 OUFR/react-next merge 
2. #14679 Move date-time components
3. #13384 uifabric/fluentui package renames
4. #14689 set up beta releases

This will be done across multiple PRs, and some of the work can be split up between multiple people.

- [x] Merge OUFR into react (1 person)
- [ ] Merge react-next into react (last chance to review FC changes). This can be split into groups of components to be done by different people.
- [x] Delete OUFR and update code references to react-next
- [ ] Either delete react-next or make it into a pure re-export package from react (and bump version to 9-alpha)
- [ ] Update gitignore so that removed files don't come back in a bad merge
- [ ] Update code references to react-next
- [ ] Delete "next" versions of vr/perf tests

### PR DESCRIPTION
**Pull request by ecraig12345**
Some final cleanup of OUFR rename and react-next merge:
- Delete the remaining v8 files (everything that's not a re-export) from react-next, and fix some re-export paths to point to react (not react-internal)
- Delete remaining react-next examples and example snapshots (merging any unique changes back into react)
- De-dupe FocusZone examples
- Make the `@fluentui/react` storybook site include current-version examples from re-exported sub-packages, for a better dev experience (you can still run storybook for individual packages too).
- Delete change files for react-examples now that we're not publishing it
- Fix `componentUrl` in doc info for components that were moved to react-internal (not react)
- Get rid of unnecessary "next" tests from perf-tests and vr-tests (and bundle size tests), and rename compat button-related tests
- Enable lint for vr-tests (it was accidentally not enabled)
- Update codeowners: remove react-next entries, fix entries for components in react-internal, and organize

Closes #15152!

### CODE REVIEW

File: packages/react-examples/.storybook/preview-loader.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
-import * as fs from 'fs';
+import * as fs from 'fs-extra';
 import * as path from 'path';
 import * as webpack from 'webpack';
 
 /**
  * Replace the PACKAGE_NAME token in preview.js with the actual current package name
  * (or error if the package doesn't have examples).
+ *
+ * Also if this is the storybook for `@fluentui/react`, replace the REACT_DEPS token
```
Comment (ecraig12345) [inline packages/react-examples/.storybook/preview-loader.js:10]: This is the interesting part for making FUIR storybook show re-exported stuff :)

File: packages/react-examples/src/react/Keytip/Keytips.Basic.Example.tsx [hunk 2]
```diff
@@ -19,36 +20,42 @@ const sampleOptions = [
...
-          <Toggle onText="Yes" offText="No" keytipProps={keytipMap.ToggleKeytip} />
-          <span>
-            Go to{' '}
-            <Link keytipProps={keytipMap.LinkKeytip} href="http://www.bing.com" target="_blank">
-              Bing
-            </Link>
-          </span>
-        </Stack>
-      </PivotItem>
+export const KeytipsBasicExample: React.FunctionComponent = () => {
```
Comment (ecraig12345) [inline packages/react-examples/src/react/Keytip/Keytips.Basic.Example.tsx:23]: @xugao Can you confirm that these example changes look correct? (they were copied from react-next)

Comment (xugao) [reply packages/react-examples/src/react/Keytip/Keytips.Basic.Example.tsx:23]: yes this is correct

**Comment by xugao** [comment]
can remove compat folder from `react-next` too

**Comment by xugao** [comment]
i still see few places using react-next:
image

**Comment by ecraig12345** [comment]
> i still see few places using react-next:

Oh good catch. Fixed now.

**Review by xugao — APPROVED** [review]
let's remove `react-next` bundle size tests for now 
```
buildEntries(
  '@fluentui/react-next',
  entries,
  false /* do not include stats for better performance. */,
  true /* onlyOwnComponents */,
);
```

**Comment by ecraig12345** [comment]
I noticed while working on this that we'd temporarily lost the codeowners for stuff that moved to react-internal. That's fixed now!

**Pull request merged**